### PR TITLE
Implement async rendering

### PR DESCRIPTION
**Pull request by dadaa**
Implement async rendering in order to begin to show the issues asap.

### CODE REVIEW

**Comment by dadaa** [comment]
I attach two videos of testing with `reddit.com` which has 3223 subnodes under `body` element.
The first one has not applied this change (performance-sync.gif)
performance-sync

On the other hand, the second one has applied those(performance-async.gif).
performance-async

The difference is:
The sync edition shows all issues after analyzing all at one go.
On the other hand, the async edition analyses and shows subnodes one by one.
Total time might be not so different (or might take a longer time a bit?) though, the experience would be better, I think.

File: extension/sidebar-pane.js [hunk 1]
```diff
@@ -16,148 +16,112 @@ async function _update() {
...
-    const { attributes, nodeName } = node;
-    issues.push(
-      ...
-       _webcompat.getHTMLElementIssues(nodeName, attributes, _targetBrowsers)
-                 .map(issue => {
-                   issue.node = node;
-                   return issue;
-                 })
-    )
+async function _asyncRenderNodes(index, nodes, skipPseudo, listEl, groupsCache) {
```
Comment (captainbrosset) [inline extension/sidebar-pane.js]: I think renaming this function to something like `_recursiveSubTreeIssuesRendering` might help make it more easy to understand.

Also, please add a short doc comment before this function to explain what it does.

**Review by captainbrosset — APPROVED** [review]
This seems sane to me. And I agree that the perceived user experience is better.

File: extension/sidebar-pane.js (hunk 1 above)
Comment (dadaa) [reply extension/sidebar-pane.js]: Okay, thank you so much.
Since we use for `selectedNode` as well, let me rename to `_recursiveNodesIssuesRendering`.

**Pull request merged**